src/Uno.Extensions.Toolkit.UI/HostBuilderExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Uno.Extensions.Toolkit.UI
```

Comment:
```suggestion
﻿namespace Uno.Extensions.Toolkit
```
keep the namespace without the .UI